src/clj/game/cards/programs.clj

```diff
@@ -2785,6 +2785,55 @@
                                                  (trigger-event state :corp :no-action))}])
                                 (make-run eid :archives (get-card state card)))}]})
 
+(defcard "Sneakdoor Prime A"
+         {:abilities [{:cost [:click 2]
+                       :prompt "Choose a server"
+                       :choices (req (cancellable
+                                       (->> runnable-servers
+                                            (map unknown->kw)
+                                            (filter is-remote?)
+                                            (map remote->name))))
+                       :msg "make a run on a remote server"
+                       :makes-run true
+                       :async true
+                       :effect (effect (register-events
+                                         card
+                                         [{:event :pre-successful-run
```

Comment:
It's ambiguous if this needs to check if it matches the target server or not. What do you think?

Reply:
Do you mean in case the runner was switched to another server? It's hard to tell from the card text:

> Make a run on a remote server. If successful, instead treat it as a successful run on a central server.

I think Sneakdoor Beta is checking archives but I wasn't sure if that was actually a ruling that the run had to be successful on archives.

Reply:
I know it's a dumb card, but maybe ask in one of the rules channels, see what people think?

Reply:
Wild - checking with GLC and it looks like these cards as written because they have 

> If successful,

Then the do need to match the target server for the effect to be made according to:

> 6.7.4. Many abilities that initiate a run contain a conditional ability with the trigger condition “If successful”. This means, “After the run created this way becomes successful”. 
a. If an effect that initiates a run specifies that that run must be made against a specific server or servers, any “If successful” abilities associated with that effect are tied to that server or servers. If the attacked server has changed since the run was initiated, and no longer is a server that could have been chosen for this run, the “If successful” abilities do not meet their trigger conditions.

But funnily enough `Sneakdoor Beta` has actually had it's own text changed to:

> :CLICK Run Archives. If that run would be declared successful, change the attacked server to HQ for the remainder of that run.

So technically these should check that the run succeeds on the target server but Sneakdoor Beta should be updated to no longer care about that restriction!

Reply:
Okay there's a big discussion on GLC about what's right here for current Sneakdoor Beta - but I definitely think it's clear these cards should have the restriction so I'll add them.

Reply:
Cool, thank you.